apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/components/AssessmentSummary.tsx

```diff
@@ -0,0 +1,254 @@
+    (result) => result.assessmentDecision === AssessmentDecision.Hold,
+  ).length;
+  return {
+    successful,
+    unsuccessful,
+    hold,
+  };
+};
+
+const AssessmentSummary = ({
+  essentialSkills,
+  nonessentialSkills,
+  assessmentResults,
+}: AssessmentSummaryProps) => {
+  const intl = useIntl();
+
+  // determine if education requirement met, should be an array of one after filtering
+  let educationAssessmentResultDecision = AssessmentDecision.Hold;
+  const educationAssessmentResult = assessmentResults.filter(
+    (result) => result.assessmentResultType === AssessmentResultType.Education,
+  );
+  if (
+    educationAssessmentResult[0] &&
+    educationAssessmentResult[0].assessmentDecision ===
+      AssessmentDecision.Successful
+  ) {
+    educationAssessmentResultDecision = AssessmentDecision.Successful;
+  } else if (
+    educationAssessmentResult[0] &&
+    educationAssessmentResult[0].assessmentDecision ===
+      AssessmentDecision.Unsuccessful
+  ) {
+    educationAssessmentResultDecision = AssessmentDecision.Unsuccessful;
+  }
+
+  // for each skill, given the assessment results array, compute the results and generate table data
+  const essentialSkillsTableData = essentialSkills.map((skill) => {
+    const essentialSkillCalculated = skillAssessmentResultCalculator(
+      skill,
+      assessmentResults,
+    );
+    return (
+      <tr key={skill.id}>
+        <td data-h2-padding="base(x.25 0 x.25 x1)">
+          {getLocalizedName(skill.name, intl)}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.successful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.unsuccessful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {essentialSkillCalculated.hold}
+        </td>
+      </tr>
+    );
+  });
+  const nonessentialSkillsTableData = nonessentialSkills.map((skill) => {
+    const nonessentialSkillCalculated = skillAssessmentResultCalculator(
+      skill,
+      assessmentResults,
+    );
+    return (
+      <tr key={skill.id}>
+        <td data-h2-padding="base(x.25 0 x.25 x1)">
+          {getLocalizedName(skill.name, intl)}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.successful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.unsuccessful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {nonessentialSkillCalculated.hold}
+        </td>
+      </tr>
+    );
+  });
+
+  return (
+    <>
+      <table data-h2-background="base(background)">
+        <thead>
+          <tr
+            data-h2-border-bottom="base(3px solid black.20)"
+            data-h2-margin-bottom="base(x.5)"
+          >
+            <th
+              scope="col"
+              data-h2-padding="base(x.25 0 x.25 x.5)"
```

Comment:
This padding does not match the `td` padding causing an odd offset.

![Screenshot 2024-01-12 080851](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/08dac572-c337-4627-a6d5-08aebf7bfdbb)

```suggestion
              data-h2-padding="base(x.25 0 x.25 x1)"
```

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/019d9ab14ea31a6398db07bd9cc0566e8860d8bd